src/index/TextIndexReadWrite.cpp

```diff
@@ -1,9 +1,68 @@
-// Copyright 2024, University of Freiburg,
-// Chair of Algorithms and Data Structures.
-// Author: Felix Meisen (fesemeisen@outlook.de)
+
```

Comment:
Make sure that all files (the old as well as the new ones) have proper copyright notices (for some reason here they are dropped).